### PR TITLE
solutions

### PR DESCRIPTION
**Pull request by VladKli**
check result, please

### CODE REVIEW

File: string_1.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                lower += 1
+            else:
+                upper += 1
+        elif letter.isdigit():
+            digits_count += 1
+        elif letter.isspace():
+            spaces += 1
+        else:
+            punct += 1
+
```
Comment (SofiaNabyvantes) [inline string_1.py:30]: return    letters_count, lower_letters_count, upper_letters_count, digits_letters_count, symbol_letters_count, spaces_letters_count

File: string_1.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    print(f'Amount of symbols: {len(string)}, amount of letters: {letters_count}, lower register: {lower},'
+          f'upper register: {upper}, amount of digits: {digits_count}, punctuation symbols: {punct}, spaces: {spaces}')
+
+
+make_statistic(string)
```
Comment (SofiaNabyvantes) [inline string_1.py:35]: letters_count, lower_letters_count, upper_letters_count, digits_letters_count, symbol_letters_count, spaces_letters_count = make_statistic(string)

Comment (SofiaNabyvantes) [reply string_1.py:35]: print(f'Amount of symbols: {all_letters_count}, amount of letters: {letters_count}, lower register: {lower},'
          f'upper register: {upper}, amount of digits: {digits_count}, punctuation symbols: {punct}, spaces: {spaces}')

File: string_2.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Користувач вводить рядок і символ. У рядку знайти всі входження цього символу і перевести його в верхній регістр,
+# а також видалити частину рядка, починаючи з останнього входження цього символу і до кінця.
+
+string = input('Print a string, please: ')
+symbol = input('Print a symbol, please: ')
+
+
+def find_result(text, symb):
```
Comment (SofiaNabyvantes) [inline string_2.py:8]: find_by_symbol
change_to_uppare_case_by_symbol

Comment (SofiaNabyvantes) [reply string_2.py:8]: revome_by_last_symbol

File: string_3.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+text = 'Даний текст. Здійснити в цьому тексті пошук і заміну заданої фрази.'
+print(text)
+
+phrase = input('Which phrase in the text do you want to change?: ')
+changing = input('Print new phrase, please: ')
+
+updated_text = text.replace(phrase, changing)
+
+print(updated_text)
```
Comment (SofiaNabyvantes) [inline string_3.py:11]: add eof line

File: string_1.py (hunk 1 above)
Comment (VladKli) [reply string_1.py:29]: +

File: string_2.py (hunk 3 above)
Comment (VladKli) [reply string_2.py]: done

File: string_3.py (hunk 4 above)
Comment (VladKli) [reply string_3.py]: done

File: string_1.py (hunk 2 above)
Comment (VladKli) [reply string_1.py]: done

**Pull request merged**